masterlist.yaml

```
@@ -9547,6 +9547,95 @@ plugins:
+      - name: 'Kapotun ARMA NFF.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA PracticalPirate.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TAWOBA.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBD Bikini.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBDUnique Treasures.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Unique Uniques.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USMP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USSEP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA WetAndCold.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Winterhold Restored.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Wyrmstooth.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun SOS.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun Vanilla Armor Patch.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+    msg:
+      - <<: *obsolete
+        subs: [ '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612)' ]
+        condition: 'version("DWKaPoTunRace.esp", "2.0", <) and not checksum("DWKaPoTunRace.esp", 0x672EB2D2)'
```

I have two questions here: how is the version metadata written in your version's description, `2.0` or `2.0.0` (i.e., with two release identifiers or three)? It shouldn't matter for how LOOT evaluates the function, but I'd like to be consistent both within the masterlist & with the mod itself, so either this needs to be changed to `2.0.0`, or the incompatibilities need to be changed to `2.0` depending on how you've written it.

The second question is that with `not checksum("DWKaPoTunRace.esp", 0x672EB2D2)`, you presumably don't want to display this message if the user has opted to us the original version? That's fine, I just want to clarify your intention here. Also, checksums in `checksum()` functions shouldn't have the `0x` prepended to them, that's only in cleaning data.
```suggestion
        condition: 'version("DWKaPoTunRace.esp", "2.0", <) and not checksum("DWKaPoTunRace.esp", 672EB2D2)'
```

---

In esp it is written as 2.0, but if LOOT treats 2.0 and 2.0.0 as the same then 2.0.0 makes more sense. As for the check, yes, it should be shown only on my version, since the two are way too different (even if my is technically superior in some functionality) and update is mandatory only for my version (fixes and simplifications galore). The rest is up to individual users to decide. I plan to provide patches fixing and adding functionality from my mod to original to keep two kinda on the same-ish level if it's not possible to update mid-game, which I'm not sure about, since most FormIDs are different.

---

Okay, thanks for clarifying; the condition looks good then. Also, versions should match how they're written in the plugin, so `2.0` would actually be preferable.